src/Microsoft.Performance.SDK/Processing/IProcessingOptionsResolver.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Microsoft.Performance.SDK.Processing.DataSourceGrouping;
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.Performance.SDK.Processing
+{
+    /// <summary>
+    ///     Responsible for choosing the <see cref="ProcessorOptions"/> that will be used for processing.
+    /// </summary>
+    public interface IProcessingOptionsResolver
+    {
+        /// <summary>
+        ///     Get the processor options for a group of data sources processed by a given processor.
```

Comment:
The summary and param tags in this docstring can use `see` tags in multiple places